cli/lib/output-filter.ts

```diff
@@ -58,6 +58,14 @@ export function removeHydraTriples({ subject, predicate, object }: Quad): boolea
   return !(isHydraTerm(subject) || isHydraTerm(predicate) || isHydraTerm(object))
 }
 
+function isOriginalValueTerm(term: Term): boolean {
+  return term.termType === 'NamedNode' && term.value.endsWith('#_original')
```

Comment:
I don't like this check, but I don't really know if it's possible to check which predicate has type `cc:OriginalValue` in a streaming way.

Reply:
Ah, this is to remove the values when publishing?

You could remove the `removeMappingsOriginalValues` step and filter out the originals when querying the observations by adding in `loadCube.ts`

```spraql
filter(not exists{    
  ?p a cc:OriginalValue    
})
```